src/main/java/com/revature/aspects/LoggingAspect.java

```diff
@@ -106,4 +128,10 @@ private String getPayload() {
 		}
 		return builder.toString();
 	}
+//	private String getResponse(@SuppressWarnings("rawtypes") ResponseEntity responseEntity) {
```

Comment:
Commented out code can be removed.

Reply:
Valid. Fixing this and 1 other logging related thing (adding the log4j2.xml file to the testing folder)